k8s/environments/stg/common-overlay/flux-patch.yaml

```diff
@@ -102,3 +102,13 @@ spec:
   values:
     java:
       image: sdshmctspublic.azurecr.io/toffee/recipe-backend:prod-68644af-20210805161941
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: hmi-case-hq-emulator
+  namespace: hmi
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/hmi/case-hq-emulator:latest
```

Comment:
```
$ docker pull sdshmctspublic.azurecr.io/hmi/case-hq-emulator
Using default tag: latest
Error response from daemon: manifest for sdshmctspublic.azurecr.io/hmi/case-hq-emulator:latest not found: manifest unknown: manifest tagged by "latest" is not found
```
🚫 blocker

Reply:
This is now pushing to acr, but PR is failing due to this missing config :)
https://github.com/hmcts/hmi-case-hq-emulator/pull/2